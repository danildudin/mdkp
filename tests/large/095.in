50 24 0
561 599 570 641 457 757 955 850 99 747 43 455 410 576 69 463 841 14 844 132 408 391 949 662 12 728 715 94 681 620 171 783 808 565 470 712 245 126 895 788 887 344 102 889 407 361 633 196 633 392 
938 938 581 564 962 207 352 723 59 770 626 193 110 989 318 4 106 923 761 939 489 622 318 90 478 642 539 782 195 264 839 179 581 573 425 86 639 209 938 936 93 705 654 265 795 888 675 852 406 214 
689 947 638 836 177 944 115 484 29 970 749 815 219 385 895 379 49 118 536 38 483 222 234 910 85 111 898 46 569 165 797 815 810 769 41 952 180 311 912 833 926 369 889 610 998 759 312 547 543 661 
479 727 399 150 501 761 628 839 413 34 356 937 880 134 871 375 283 964 917 156 790 807 907 90 315 373 541 501 849 568 438 144 577 883 43 101 373 261 8 861 635 685 597 552 547 365 80 204 478 394 
37 937 176 522 838 714 236 292 226 982 903 910 848 408 109 503 547 128 136 373 685 144 136 383 656 366 904 429 95 816 740 309 799 967 256 973 577 543 531 829 828 275 991 894 221 958 521 307 271 177 
845 738 233 585 213 932 693 245 276 706 434 273 239 673 654 827 75 152 818 978 819 10 368 432 344 721 613 303 317 811 377 734 541 998 131 433 772 794 656 159 572 939 58 778 838 98 153 494 159 44 
190 692 268 894 160 844 563 50 452 163 856 590 33 315 168 430 940 81 320 358 625 485 905 318 508 497 222 8 28 644 656 653 604 393 599 753 251 892 479 669 187 741 696 559 44 549 201 787 22 3 
751 697 342 148 785 349 824 790 201 316 282 205 197 499 606 832 492 19 49 353 391 354 773 369 340 144 796 158 847 747 618 658 405 702 777 973 449 646 684 507 357 862 529 602 938 689 431 967 696 165 
259 742 306 931 933 478 668 544 860 755 241 29 758 664 411 453 453 760 207 780 726 486 413 980 191 177 66 417 572 91 269 886 15 886 713 662 187 596 470 648 638 382 823 996 803 221 15 519 718 134 
634 835 777 676 440 555 682 730 387 790 45 234 740 135 287 933 168 612 921 452 964 762 607 207 976 861 46 759 798 975 320 495 24 929 704 944 509 575 828 214 64 486 937 647 185 179 868 178 451 934 
891 47 370 512 726 905 987 423 997 28 231 293 888 803 486 458 733 615 162 203 224 199 262 164 471 975 588 902 255 221 503 880 285 329 445 854 617 905 790 96 463 935 534 180 980 13 427 584 554 681 
7 850 441 478 170 400 753 366 93 133 981 290 65 581 855 793 467 67 462 387 181 62 461 260 333 944 736 826 713 581 60 352 419 764 682 843 406 40 356 817 284 202 157 951 74 188 442 487 628 994 
614 607 147 953 405 433 580 723 663 208 719 806 447 326 108 306 291 433 820 375 438 860 616 282 398 631 831 446 400 893 9 359 1000 912 38 199 728 648 834 518 627 3 686 815 473 592 400 374 718 174 
765 78 523 647 462 899 176 588 181 667 970 758 429 761 358 876 389 158 578 512 325 635 62 132 583 312 312 9 561 196 556 488 639 228 305 519 923 791 794 836 600 809 298 914 994 64 606 571 915 154 
47 866 221 777 605 655 482 629 236 76 701 411 310 130 770 973 277 214 553 773 336 566 507 619 557 62 261 877 430 960 213 749 300 117 256 721 645 980 748 437 271 741 598 843 150 726 496 186 299 80 
672 673 118 804 37 485 487 265 31 713 627 869 272 958 126 552 374 954 253 212 177 239 204 844 744 212 987 133 442 291 806 360 116 322 537 959 68 999 241 952 274 256 246 911 207 541 372 737 110 54 
730 656 52 437 669 123 502 544 199 603 705 523 241 800 881 865 730 430 170 786 224 380 493 132 655 106 822 658 913 508 206 418 15 636 109 878 115 53 609 43 564 276 712 800 238 677 818 820 90 196 
355 41 237 691 747 739 723 789 784 484 752 899 521 204 519 774 365 187 208 891 431 315 514 503 865 162 867 771 759 352 684 675 312 35 78 563 234 880 884 155 702 406 450 935 431 338 321 186 461 109 
991 759 49 842 373 57 454 173 778 783 889 147 202 544 316 387 48 890 834 16 391 986 759 359 770 685 751 645 668 406 100 9 335 500 580 30 435 417 443 768 883 422 660 887 289 5 338 853 982 135 
774 305 632 504 924 661 85 883 190 533 303 38 287 16 135 605 145 547 817 434 98 559 219 191 833 477 893 153 649 566 274 781 74 776 436 347 434 743 290 39 462 350 226 936 111 442 624 41 354 418 
128 761 298 303 175 104 328 242 910 298 346 902 36 220 640 221 871 920 137 260 430 930 605 197 909 611 330 717 33 330 752 11 462 371 90 812 888 780 525 914 251 79 625 457 128 895 263 369 379 412 
640 565 839 395 240 903 884 417 577 129 741 76 792 770 224 49 9 362 651 20 678 761 845 756 705 102 417 902 404 936 786 44 774 698 381 41 730 984 510 156 365 740 696 895 958 338 291 434 959 539 
683 499 459 204 979 480 743 462 146 875 389 966 285 921 915 336 543 721 585 580 349 206 552 540 400 795 572 704 646 821 941 1 165 745 46 794 212 609 435 227 861 233 891 402 871 383 412 112 17 68 
641 451 591 195 409 97 382 352 585 570 389 552 363 314 458 121 427 564 306 906 104 269 439 709 675 137 689 10 86 148 449 19 709 825 131 513 835 59 195 559 20 120 571 766 547 773 979 168 131 724 
198 441 399 531 865 975 458 439 553 299 17 409 456 344 996 155 458 826 760 74 876 86 92 834 670 459 770 421 939 290 506 624 990 102 265 294 65 260 269 847 952 353 809 881 557 537 527 374 960 938 
50539 51909 30231 99244 62258 62513 15426 29509 6002 29236 65917 67788 25513 87867 39017 1839 43309 68896 99869 37181 74627 61001 20933 18988 
