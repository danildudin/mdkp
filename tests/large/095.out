7998
11
2 5 6 18 23 25 31 32 34 39 48 
